assets/src/edit-story/app/rightClickMenu/constants.js

```diff
@@ -33,16 +33,20 @@ export const RIGHT_CLICK_MENU_LABELS = {
   BRING_TO_FRONT: __('Bring to Front', 'web-stories'),
   CLEAR_IMAGE_STYLES: __('Clear Image Styles', 'web-stories'),
   COPY_IMAGE_STYLES: __('Copy Image Styles', 'web-stories'),
+  COPY_STYLES: __('Copy Style', 'web-stories'),
   COPY: __('Copy', 'web-stories'),
   DELETE_PAGE: __('Delete Page', 'web-stories'),
   DELETE: __('Delete', 'web-stories'),
   DUPLICATE_PAGE: __('Duplicate Page', 'web-stories'),
+  PASTE_STYLES: __('Paste Style', 'web-stories'),
   PASTE_IMAGE_STYLES: __('Paste Image Styles', 'web-stories'),
   PASTE: __('Paste', 'web-stories'),
   SCALE_AND_CROP_IMAGE: __('Scale & Crop Image', 'web-stories'),
   SEND_BACKWARD: __('Send Backward', 'web-stories'),
   SEND_TO_BACK: __('Send to Back', 'web-stories'),
   SET_AS_PAGE_BACKGROUND: __('Set as Page Background', 'web-stories'),
+  ADD_TO_TEXT_PRESETS: __('Add style to “Saved Styles”', 'web-stories'),
+  ADD_TO_COLOR_PRESETS: __('Add color to “Saved Colors”', 'web-stories'),
```

Comment:
This isn't documented, but we're keeping everything in title case. The figma designs aren't up to date here

```suggestion
  ADD_TO_TEXT_PRESETS: __('Add Style to “Saved Styles”', 'web-stories'),
  ADD_TO_COLOR_PRESETS: __('Add Color to “Saved Colors”', 'web-stories'),
```